doc/advanced-configuration/Listener-modules.md

```diff
@@ -99,6 +99,10 @@ Unlike `ejabberd_c2s`, it doesn't use `ejabberd_receiver` or `ejabberd_listener`
 ### HTTP module: `mod_cowboy`
 
 This module provides an additional routing layer on top of HTTP(s) or WS(S) protocols.
+Regular cowboy handlers cannot cleanly handle both protocols on the same port. 
```

Comment:
It is "Cowboy" with an uppercase "C".

Reply:
IMO, it doesn't have to start with the uppercase "C". It's worth surrounding it with `` though like `cowboy`.